playwright_test/locaterPackage/kmLocators.js

```diff
@@ -0,0 +1,22 @@
+
+/*
+ * kmLocators.js
+ * playwright_test/utils
+ * Created by Archit on 21/12/2011. 
+ */
+
+export const locators = {
+    envBtn : '//label[@for="test"]',
+    appIdField : '//span[text()="App ID"]/following::input',
+    scriptFiled : '//textarea[@data-gramm="false"]',
+    launchWidgetBtn : '(//button[@type="button"])[3]',
+    logoutWidgetBtn : '//button[text()="Logout & Reload"]'
+}
+
+export const widgetLocators = {
+    kmIframe : 'iframe[name="Kommunicate widget iframe"]',
+    kmLaunchWidget : '//*[@id="launcher-svg-container"]',
+    kmTextBox : '#mck-text-box',
+    kmSendButton : '//button[@title="Send Message"]//span[1]',
+    kmMsgStatus : '//*[@id="mck-message-cell"]/div[2]/div[1]/div/div[1]/div[2]/div[2]/div[4]/span[2]'
```

Comment:
@archit2602 chk here also has multiple divs

Reply:
ya I guess you missed the message I said I can't do anything with `kmMsgStatus` Xpath because this is the only thing I have to get that